autoload/crunch.vim

```diff
@@ -78,7 +78,10 @@ function! crunch#eval(exprs) abort "{{{2
 "    Decho '== Inizilation =='
     let s:variables = deepcopy(g:crunch_user_variables, 0)
 
-    let expr_list = split(a:exprs, '\n', 1)
+    let separator_magic = "\t   \t  \t\n"
```

Comment:
I'm going to study this a bit before the merge and see if it's possible to not use the magic separator.

Reply:
Thanks! I didn't get any better method to remember the positions of `;` and `\n`.